.github/workflows/pre-release.yml

```diff
@@ -73,8 +73,8 @@ jobs:
           
       - name: "Create archives"
         run: |
-          powershell Compress-Archive ".\" "seekers.zip"
-          powershell Compress-Archive ".\seekers\grpc\stubs" "seekers-stubs.zip"
+          powershell Compress-Archive ".\" "seekers-win32.zip"
```

Comment:
+1

Reply:
+1, if we later change later the names again.